### PR TITLE
New command 'repos add-member' to allow a user access to a repo

### PR DESCRIPTION
**Pull request by jstroem**

### CODE REVIEW

**Comment by anagrius** [comment]
Looks real good! Thanks.

File: cmd/repos_add_member.go [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+		Short: "Adds a user to a repository.",
+		Args:  cobra.ExactArgs(2),
+		Run: func(cmd *cobra.Command, args []string) {
+			repoName := args[0]
+			userName := args[1]
+
+			client := NewApiClient(cmd)
+
+			_, apiErr := client.Repositories().AddMember(repoName, userName, adminRights, deleteRights)
+			exitOnError(cmd, apiErr, "error creating repository")
```
Comment (anagrius) [inline cmd/repos_add_member.go]: Wrong error message

File: cmd/repos_add_member.go [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+		},
+	}
+
+	cmd.Flags().BoolVar(&adminRights, "admin-rights", false, "Grant the user admin rights")
```
Comment (anagrius) [inline cmd/repos_add_member.go]: @floekke How does this work with roles?

File: cmd/repos_add_member.go (hunk 1 above)
Comment (jstroem) [reply cmd/repos_add_member.go]: fixed

**Comment by anagrius** [comment]
Closing since there is no activity

**Pull request closed without merging**